tests/sentry/event_manager/test_hierarchical_hashes.py

```diff
@@ -325,6 +255,42 @@ def save_event(ts_offset):
         assert group.last_seen == event2.datetime
 
         # After expiry, new events are still assigned to the same group:
-        project.update_option("sentry:secondary_grouping_expiry", 0)
-        event3 = save_event(4)
+        self.project.update_option("sentry:secondary_grouping_expiry", 0)
+        event3 = self.save_event()
         assert event3.group_id == event2.group_id
+
+    def save_event(self):
+        manager = EventManager(
+            make_event(
+                message="foo 123",
+                event_id=hex(2**127)[-32:],
```

Comment:
Using the timestamp offset was not necessary. Two events can still have the same ID.
<img width="275" alt="image" src="https://github.com/getsentry/sentry/assets/44410/97609785-076d-4523-8633-b0c02be2b29c">